tests/test_domain_utils.py

```diff
@@ -0,0 +1,18 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""domain_utilstes extract test class."""
+
+from ..msticpy.sectools import domain_utils
+
+
+def test_validate_domain():
+    test_dom_val = domain_utils.DomainValidator()
+    valid_tld = test_dom_val.validate_tld("www.microsoft.com")
```

Comment:
I know we don't have many negative tests but is it worth putting some known broken domain to act as a negative, given the unit test is pretty simple?

Reply:
Good idea, I will add one.